content/post/on-the-color-schemes-of-open-access.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "On the color schemes of Open Access"
+date: 2020-05-07T16:00:00+02:00
+author: "Karsten Schuldt (FH Graubünden)"
+draft: false
+tags: ["open access", " analytics"]
+---
+
+A problem one encounters with Open Access (OA) is the wide range of understandings of different OA types and classifications. What started as a classification of two "colors" (gold and green, see Harnad et al. 2008[^2]) has evolved into a plethora of possible classifications. Right at the outset of the OA movement the SHERPA/RoMEO-project (2002-2006) proposed an enhanced four color scheme (green, blue, yellow, white; see Jenkis et al. 2007[^3]). Since then, the number of colors meaning different classifications of OA, have grown rapidly. E.g. one LiBGuide of the De Paul-University, Chicago, now lists seven types of OA (green, blue, yellow, gold, white, black and hybrid; see Lucic (n.d.)[^4]). Taubert et al. (2019: 3[^6]; further discussion see below) count 14 different colors used in different studies on OA.
+
+<!--more-->
+
+There are always good reasons to enhance, expand or rebuilt OA classifications. But at the same time, these constant changes create new problems as well. Van Leeuwen et al. (2017[^7]) discuss those problems for the research and analysis of OA when combining different data sources. They introduce a method to label OA publications automatically, but note limitations of this method as well. Merging data from resources with different color schemes, or just comparing them, always imposes decisions on which color schemes to follow. This holds true for repositories like SONAR, which harvest other repositories, as well as for every form of OA analysis. As is so often the case with classifications, there seems to be no perfect single one for all use cases. As a rule of thumb, the classification which suits the data model of one specific repository or which contributes the most to the concrete questions of a specific analysis should be chosen.
+
+Today, most of the analyses of OA use the color scheme of the unpaywall-project (gold, green, hybrid, bronze, closed; introduced in Piwowar et al. 2018[^5]), because most analyses use the data sets of unpaywall. (E.g. the [German OA-monitor](https://open-access-monitor.de/#/open-access) For now, this scheme seems to have the upper-hand, but is far from perfect. In this blog post, I summarize the discussions of two recent proposals, both published just last year, which both give arguments for a more granular classification of OA. The fact that there are two different proposals only shows that the discussion about all the different colors of OA is far from over.
```

Comment:
A closing bracket is missing after `(E.g. the [German OA-monitor](https://open-access-monitor.de/#/open-access)`